web/src/lib/nostr/pool.ts

```diff
@@ -231,6 +245,46 @@ export class Pool {
         })
     }
 
+    public async sendReaction(message, reaction: string) {
+        const noteId = message.id;
+        const notePubkey = message.pubkey;
+
+        console.log('******* SENDING REACTION: ' + reaction + ' to note with Id: ' + noteId + ' pubkey: ' + notePubkey);
+
+        if (reaction.length !== 1) {
+            console.error('Trying to send reactions with > 1 character is not allowed by nip-25');
+            return;
+        }
+
+        if (!noteId || !notePubkey) {
+            return false;
+        }
+
+        // 1
+        const event: Event = {
+            kind: 7,
```

Comment:
I've seen this "7" before... it should have a constant!

Reply:
That's fixed in the next PR already, using the new Enum for types.